feedback/api/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from feedback.api import views
+
+urlpatterns = [
+    path('<int:pk>/reaction', views.ReactToNews.as_view()),
+    path('<int:pk>/comment', views.CommentOnNews.as_view()),
```

Comment:
this url could be: folan/news/feedback/comment/<str:token>/ or folan/feedback/comment/<str:token>/ and the same goes for reaction url